tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -1637,6 +1637,223 @@ def test_projector_dynamic_type(max_workers, n_wires):
         assert np.isclose(res, 1 / 2**n_wires)
 
 
+@pytest.mark.all_interfaces
+@pytest.mark.parametrize("interface", ["numpy", "autograd", "torch", "jax", "tensorflow"])
```

Comment:
Just a heads up that `interface="numpy"` is exactly the same as `interface="autograd"` for the qnode.